ouroboros-consensus-test/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -91,18 +105,18 @@ prop_Mempool_snapshotTxs_snapshotTxsAfter :: TestSetup -> Property
 prop_Mempool_snapshotTxs_snapshotTxsAfter setup =
     withTestMempool setup $ \TestMempool { mempool } -> do
       let Mempool { getSnapshot } = mempool
-      MempoolSnapshot { snapshotTxs, snapshotTxsAfter} <- atomically getSnapshot
-      return $ snapshotTxs === snapshotTxsAfter zeroTicketNo
+      snap@MempoolSnapshot {snapshotTxsAfter} <- atomically getSnapshot
+      return $ snapshotTxs snap === snapshotTxsAfter zeroTicketNo
 
 -- | Test that all valid transactions added to a 'Mempool' can be retrieved
 -- afterward.
 prop_Mempool_addTxs_getTxs :: TestSetupWithTxs -> Property
 prop_Mempool_addTxs_getTxs setup =
     withTestMempool (testSetup setup) $ \TestMempool { mempool } -> do
       _ <- addTxs mempool (allTxs setup)
-      MempoolSnapshot { snapshotTxs } <- atomically $ getSnapshot mempool
+      snap <- atomically $ getSnapshot mempool
```

Comment:
This was due to some changes in the snapshot. This whole file could be reverted I think.